website/docs/feature-flag-tutorials/use-cases/ai.md

```diff
@@ -0,0 +1,223 @@
+
+```sh
+git clone https://github.com/unleash/unleash.git
+cd unleash
+docker compose up -d
+```
+
+You now have Unleash installed on your machine and running in the background. You can access this instance in your web browser at [http://localhost:4242](http://localhost:4242).
+
+Log in to the platform using these credentials:
+
+```
+Username: admin
+Password: unleash4all
+```
+
+Click **New feature flag** to create a new feature flag.
+
+![Create a new feature flag](../ruby/new-ff.png)
+
+Create a new feature flag called `gpt-version`.
+
+After that, and this the most important part, we need to add a variant to our feature flag.
+
+What we'll do is hold all the model configurations in the feature flag variants.
+
+So, we can take these model configurations:
+
+```javascript
+const variants = {
+    variant1: {
+        model: "gpt-3.5-turbo",
+        max_tokens: 4096,
+        temperature: 1,
+    },
+    variant2: {
+        model: "gpt-4-turbo",
+        max_tokens: 4096,
+        temperature: 1,
+    },
+    variant3: {
+        model: "gpt-4-vision-preview",
+        max_tokens: 4096,
+        temperature: 1,
+    },
+};
+```
+
+And add them into Unleash.
+
+![a variant with parameters for an OpenAI model](./model-variant.png)
+
+## Querying AI feature flags
+
+Now let's go back to the code and grab our AI config from the feature flag that we just created.
+
+First, install the Unleash Node.js client:
+
+```sh
+npm install unleash-client
+```
+
+Now, let's modify our `+server.js` file to use Unleash:
+
+```javascript
+import { initialize } from "unleash-client";
+import { createOpenAI } from "@ai-sdk/openai";
+import { streamText } from "ai";
+import { env } from "$env/dynamic/private";
+
+const openai = createOpenAI({
+    apiKey: env.OPENAI_API_KEY ?? "",
+});
+
+const unleash = initialize({
+    url: "http://localhost:4242/api/",
+    appName: "my-ai-app",
+    customHeaders: { Authorization: env.UNLEASH_API_KEY ?? "" },
+});
+
+export const POST = async ({ request }) => {
+    const { messages } = await request.json();
+
+    // Get the feature flag variant
+    const variant = unleash.getVariant("gpt-versions");
+
+    const result = await streamText({
+        model: openai(variant.model),
+        messages,
+        max_tokens: variant.max_tokens,
+        temperature: variant.temperature,
+    });
+
+    return result.toDataStreamResponse();
+};
+```
+
+This setup uses the Unleash client to fetch the value of a feature flag called `gpt-versions`.
+
+Now, instead of hardcoding 'variant1', we're dynamically choosing the AI model based on the feature flag variant.
+
+This setup gives us a ton of flexibility. Want to roll out GPT-4 to 10% of your users? Easy. Need to quickly switch everyone back to GPT-3.5 because of a bug? No problem. You can do all of this from your Unleash dashboard without touching your code.
+
+## Conclusion
+
+Thanks for following along! You've learned one way to use feature flags to interact with AI models.
```

Comment:
The conclusion feels a bit lacking. We might want to do a quick recap of what we’ve covered in this guide and how it can help readers going forward.

Here's a suggestion:

Thanks for sticking with us! In this guide, we covered how to use feature flags to manage AI models more flexibly. By integrating feature flags, you can quickly switch between different model configurations, experiment with variations, and even roll out updates without needing to touch your code or redeploy. This gives you more control over how and when your AI features are used, whether you’re rolling out a new model or responding to something unexpected, like running out of credits or discovering a bug.

Hopefully, this gives you some useful tools to better manage your AI experiments, allowing you to adapt quickly and keep things running smoothly.